scripts/ci/test-pr

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
+
+set -e
+
+# Runs ci/test-operator on each operator with changes in a PR.
+DIR="$(cd "$(dirname ${BASH_SOURCE[0]})" && pwd)"
+OP_TYPES=( "upstream-community-operators" )
+for op_type in ${OP_TYPES[@]}; do
+  for file in $(git diff master --name-only 2>&1); do
```

Comment:
Does this include files that were removed from the test branch? If so, how can we avoid those files?

Reply:
No, I'll remove those from the returned results.

Reply:
On second thought, we should test a PR that removes files as long as the operator package still exists.